src/ddmd/dimport.d

```diff
@@ -304,8 +304,13 @@ extern (C++) final class Import : Dsymbol
             {
                 AliasDeclaration ad = aliasdecls[i];
                 //printf("\tImport %s alias %s = %s, scope = %p\n", toPrettyChars(), aliases[i].toChars(), names[i].toChars(), ad._scope);
-                if (mod.search(loc, names[i]))
+
+                Dsymbol importedSymbol = mod.search(loc, names[i]);
+                if (importedSymbol)
                 {
+                    // This deprecation should turn into an error as soon as possible
+                    if (importedSymbol.prot().kind == PROTprivate)
```

Comment:
I think it would be better if you used one of these functions.

https://github.com/dlang/dmd/blob/8d6efa5dd461ac9c28caf1964a9dab2ec86232b3/src/dmd/access.d#L481-L525

Reply:
Either:
```
if (!symbolIsVisible(mod, importedSymbol) { error; }
```
or
```
if (!symbolIsVisible(sc, importedSymbol) { error; }
```